docs/API-Reference.md

```diff
@@ -5228,6 +5229,42 @@ This event occurs when an app has a new version and is now available for request
 }
 ```
 
+### Pipeline Promotion Event Payload
+
+This event occurs upon a successful pipeline promotion. It fires on the target apps in the coupling, and includes information about the source app (`promoted_from`) and the new release (`release`).
+
+#### Target App
+
+`POST [callback end point]`
+
+```json
+{
+  "action":"pipeline_promotion",
+  "app":{
+    "name":"targetapp",
+    "id":"7edbac4b-6a5e-09e1-ef3a-08084a904621"
+  },
+  "space":{
+    "name":"target-space"
+  },
+  "promoted_from":{
+    "app":{
+      "name":"sourceapp",
+      "id":"7edbac4b-6a5e-09e1-ef3a-08084a904621"
+    },
+    "space":{
+      "name":"source-space"
+    }
+  },
+  "release":{
+    "id":"1edbac4b-6a5e-09e1-ef3a-08084a904623",
+    "status":"queued",
+    "created_at":"2016-08-09T12:00:00Z",
+    "version":13,
+    "description":"Promotion from sourceapp-source-space"
+  }
+}
+```
 
```

Comment:
At the top of this doc section there's a list of all of the events, can you add this new event to that list?

Reply:
Nevermind, I just realized you did. Sorry